src/sasl_anonymous.c

```diff
@@ -112,17 +106,27 @@ int saslanonymous_challenge(CONCRETE_SASL_MECHANISM_HANDLE concrete_sasl_mechani
 	}
 	else
 	{
-		/* Codes_SRS_SASL_ANONYMOUS_01_013: [saslanonymous_challenge shall set the response_bytes buffer to NULL and 0 size as the ANONYMOUS SASL mechanism does not implement challenge/response.] */
+		/* Codes_SRS_SASL_ANONYMOUS_01_013: [`saslanonymous_challenge` shall set the `buffer` field to NULL and `size` to 0 in the `response_bytes` argument as the ANONYMOUS SASL mechanism does not implement challenge/response.] */
 		response_bytes->bytes = NULL;
```

Comment:
We're talking about replacement of Codes_01_013 with Codes_01_013 the new one. I do not understand what you thing is removed ...

---
In reply to: [111764122](https://github.com/Azure/azure-uamqp-c/pull/93#discussion_r111764122) [](ancestors = 111764122)